src/types/fetchers.ts

```diff
@@ -82,9 +82,9 @@ export enum FetcherName {
   OPTIONS_PRICE = 'OptionsPrice',
   YEARN_VAULT_TOKEN_PRICE = 'YearnVaultTokenPrice',
   RANDOM_NUMBER = 'RandomNumber',
-  SOVRYN_PRICE = 'SovrynPriceFetcher',
-  BY_BIT = 'ByBit',
-  BINANCE = 'Binance',
+  SOVRYN_PRICE = 'SovrynPrice',
```

Comment:
```suggestion
  SovrynPrice = 'SovrynPrice',
```
I think we should use correct name also as key